tests/test_filters.py

```diff
@@ -1,54 +1,31 @@
 # -*- coding: utf-8 -*-
-from __future__ import absolute_import
-from __future__ import unicode_literals
+from __future__ import absolute_import, unicode_literals
 
-from collections import OrderedDict
-from datetime import date, time, timedelta, datetime
 import inspect
-import mock
+from collections import OrderedDict
+from datetime import date, datetime, time, timedelta
 
+import mock
 from django import forms
 from django.test import TestCase, override_settings
 from django.utils import translation
 from django.utils.translation import ugettext as _
-
 from django_filters import filters, widgets
-from django_filters.fields import (
-    Lookup,
-    RangeField,
-    DateRangeField,
-    DateTimeRangeField,
-    TimeRangeField,
-    LookupTypeField,
-    BaseCSVField)
-from django_filters.filters import (
-    Filter,
-    CharFilter,
-    BooleanFilter,
-    ChoiceFilter,
-    MultipleChoiceFilter,
-    TypedMultipleChoiceFilter,
-    DateFilter,
-    DateTimeFilter,
-    TimeFilter,
-    DurationFilter,
-    ModelChoiceFilter,
-    ModelMultipleChoiceFilter,
-    NumberFilter,
-    NumericRangeFilter,
-    RangeFilter,
-    DateRangeFilter,
-    DateFromToRangeFilter,
-    DateTimeFromToRangeFilter,
-    TimeRangeFilter,
-    AllValuesFilter,
-    BaseCSVFilter,
-    BaseInFilter,
-    BaseRangeFilter,
-    UUIDFilter,
-    OrderingFilter,
-    LOOKUP_TYPES)
-
+from django_filters.fields import (BaseCSVField, DateRangeField,
+                                   DateTimeRangeField, Lookup, LookupTypeField,
+                                   RangeField, TimeRangeField)
+from django_filters.filters import (LOOKUP_TYPES, AllValuesFilter,
+                                    BaseCSVFilter, BaseInFilter,
+                                    BaseRangeFilter, BooleanFilter, CharFilter,
+                                    ChoiceFilter, DateFilter,
+                                    DateFromToRangeFilter, DateRangeFilter,
+                                    DateTimeFilter, DateTimeFromToRangeFilter,
+                                    DurationFilter, Filter, ModelChoiceFilter,
+                                    ModelMultipleChoiceFilter,
+                                    MultipleChoiceFilter, NumberFilter,
+                                    NumericRangeFilter, OrderingFilter,
+                                    RangeFilter, TimeFilter, TimeRangeFilter,
+                                    TypedMultipleChoiceFilter, UUIDFilter)
```

Comment:
This is the one I don't like. 

(One per line with a single indent by preference.)